### PR TITLE
Update stable/dask chart to v3.0.0

### PR DESCRIPTION
**Pull request by rmccorm4**
#### What this PR does / why we need it:
1. Update API versions to v1
2. Refactor NOTES.txt to handle differnt serviceTypes
3. Create OWNERS file
4. Add comments for possible serviceType and GPU values in values.yaml
5. Update README values and add some new sections for serviceTypes and Jupyter password
6. Lint README, yaml files, and rendered yaml templates
7. Update container images to v1.1.1
8. Update Chart Version to 3.0.0

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/dask/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+- danielfrg
+- mrocklin
+- beberg
+reviewers:
+- danielfrg
+- mrocklin
+- beberg
```
Comment (beberg) [inline stable/dask/OWNERS]: Editor ate the end of file linefeeds, please check all files.

File: stable/dask/values.yaml [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
 scheduler:
   name: scheduler
   image:
     repository: "daskdev/dask"
-    tag: 1.1.0
+    tag: 1.1.1
     pullPolicy: IfNotPresent
   replicas: 1
+  # serviceType: "ClusterIP"
```
Comment (beberg) [inline stable/dask/values.yaml:15]: If the user removes the "#" characters valid YAML should remain, so take out the spaces after the "#" here. Other comments look good but check.

File: stable/dask/README.md [hunk 3]
```diff
@@ -2,38 +2,41 @@
...
+for the differences between ClusterIP, NodePort, and LoadBalancer.
 
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-release stable/dask
+helm install --name my-release stable/dask
 ```
```
Comment (beberg) [inline stable/dask/README.md:27]: Add after the command: "Depending on how your cluster was setup, you may also need to specify --namespace my-namespace"

**Comment by rmccorm4** [comment]
1. Added nodeSelector and imagePullSecrets to templates and values.yaml
2. Tested `env` variables like `EXTRA_CONDA_PACKAGES` in jupyter deployment.
3. Tested new imagePullSecrets with/without a secret for a dummy private Docker container on dask-worker deployment.
    - Without valid secret: Pods gets ImagePullBackoff error and terminates
    - With valid secret: Pod sucessfully pulls container (but gets runtime error from trying to execute `dask-worker` command as expected)

**Pull request closed without merging**